docker/Dockerfile

```diff
@@ -22,9 +22,12 @@ ARG BESU_USER=besu
 USER ${BESU_USER}
 WORKDIR /opt/besu
 
-ADD --chown=besu:besu https://github.com/grafana/pyroscope-java/releases/download/v0.18.1/pyroscope.jar /opt/pyroscope/lib/pyroscope.jar
+# support for pyroscope
+
 
 COPY --chown=besu:besu besu /opt/besu/
+ADD --chown=besu:besu https://github.com/grafana/pyroscope-java/releases/download/v0.18.1/pyroscope.jar /opt/besu/pyroscope/pyroscope.jar
```

Comment:
Why not use the last release : v2.1.2 https://github.com/grafana/pyroscope-java/releases/download/v2.1.2/pyroscope.jar

Reply:
Nice, thanks @ahamlat  didn't know there was a new one. Good to go - have upped to 2.1.2